site/src/components/WorkspaceScheduleButton/WorkspaceScheduleButton.tsx

```diff
@@ -0,0 +1,249 @@
+import Button from "@material-ui/core/Button"
```

Comment:
Can we break this file up a bit? It's 250 lines.

Reply:
Yes, I usually like to keep things on the same file if they are only used there, it helps me to identify things that are used by multiple components or not, but I see your point. I will move this component to a diff file and some of these functions to a utils file too.

Reply:
I hear you, I also dislike clicking around. Maybe this would be a good case to colocate stuff in the same folder so it's not too far away, but also isn't all in the same file.

Reply:
Agreed, done!